code/controllers/subsystem/ping.dm

```diff
@@ -0,0 +1,32 @@
+SUBSYSTEM_DEF(ping)
+	name = "Ping"
+	priority = FIRE_PRIORITY_PING
+	wait = 30  // one and a half seconds
```

Comment:
Wait is in deciseconds, not ticks, unless it has the ticker subsystem flag (this does not).

This is 3 seconds of wait.